ui/admin/app/routes/scopes/scope/app-tokens/app-token.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+
+export default class ScopesScopeAppTokensTokenRoute extends Route {
+  @service store;
+
+  async model(params) {
+    // Fetch the app-token details using the token id and scope id
+    return this.store.findRecord('app-token', params.token_id, {
```

Comment:
can we specify the `reload: true` options so that a get request is made even if the record is stashed in the ember store.